CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 1.0.0
+* Added support for load snapshot. (#414)
+* Added support for set drive IO engine type. (#411) 
+* Added support for get Firecracker version. (#410)
+* Adds a DriveOpt func for configuring CacheType. (#401)
+* Adds GetExportVMConfig to read the /vm/config endpoint. (#400)
+* Adds ability to allow passing cgroup-version to jailer. (#399)
+* Adds support for the MMDS configuration. (#290)
+* Adds PauseVM, ResumeVM, and CreateSnapshot capabilities. (#278)
+* Fixes InitrdPath usage. (#295)
```

Comment:
A few comments I had:

1. We should settle on a single tense. I prefer past e.g. "Added" over "Adds" Thoughts?
2. Should we group like features? e.g. Added support for create and load snapshot (#278, #414)

Reply:
1.) I do think that makes sense. I just wanted to be more in-line with previous change logs in older releases as they also use present tense. 

2.) I agree. That is more comprehensive.

Reply:
Is the consensus that we use past tense for the changelog?